c4/README.md

```diff
@@ -0,0 +1,28 @@
+# Quiet C4 Diagrams
+
+Quiet architecture presented on C4 diagrams. Currently it uses 2 diagrams - Context and Container.
+
+Made in Structurzir
+
+Current views saved in `/current`
+
+## Run locally
+
+1. Make sure you have Docker installed
+2. Run command in terminal: ```docker pull structurizr/lite```
+3. Run docker container inside your Quiet directory and choosen porsts, example : ```docker run -it --rm -p 8080:8080 -v /home/kacper/development/quiet:/usr/local/structurizr structurizr/lite```
```

Comment:
I guess it'd be good to replace `/home/kacper/developer/quiet` with something like `/__PATH_TO_REPOSITORY__`